test/js/node/process/stdin/run-with-pty-readable.py

```
@@ -0,0 +1,71 @@
+        if ready:
+            data = os.read(master_fd, 1024)
+            if data:
+                buffer += data
+                sys.stdout.buffer.write(data)
+                sys.stdout.buffer.flush()
+
+def waitAndWrite(b):
+    waitForReady()
+    os.write(master_fd, b)
+
+def timeout_handler(signum, frame):
```

Ruff (0.15.9)

[warning] Unused function argument: `signum`

(ARG001)